package.json

```diff
@@ -63,6 +63,7 @@
     "grunt-exec": "0.4.6",
     "grunt-jscs": "2.1.0",
     "grunt-mocha-cli": "1.14.0",
+    "grunt-cli": "0.1.13",
```

Comment:
I just wanted the install process to be slightly less painful. If there is no way to indicate in package.json that this package needs to be installed globally this prerequisite needs to be added to the README.